src/Entities/Behaviors/HasUuid.php

```diff
@@ -11,7 +11,6 @@
      *
      * @ORM\Id
      * @ORM\Column(type="uuid")
-     * @ORM\GeneratedValue(strategy="UUID")
```

Comment:
Is this correct?

Reply:
Probably not but I need to come back to in a second. Doctrine threw a fit so I'm generating the IDs in the hydrator for the moment.

Reply:
I think that's fine. AFAIK we're planning to do that anyway, so it should be all good to do away with this...